mlrun/feature_store/feature_set.py

```diff
@@ -1277,18 +1291,76 @@ def do(self, event):
                 for window in windows:
                     spark_window = self._duration_to_spark_format(window)
                     aggs = last_value_aggs
+                    all_aggs_names = ""
                     for operation in operations:
                         agg = self._get_aggr(operation, column)
                         agg_name = f"{name if name else column}_{operation}_{window}"
                         agg = agg.alias(agg_name)
                         aggs.append(agg)
+                        all_aggs_names = all_aggs_names + agg_name
```

Comment:
No separator between aggregation names?

Reply:
The idea is to create a unique folder within the checkpoint directory, like {checkpointdir}/{project_name}/{featureset_name}/{all_aggs_names}. Thats why I remove all spaces and other non alphabetical and numerical characters from all_aggs_names